hub/config/src/test/java/uk/gov/ida/hub/config/domain/builders/IdentityProviderConfigDataBuilder.java

```diff
@@ -5,16 +5,13 @@
 import org.joda.time.DateTime;
 import uk.gov.ida.hub.config.domain.IdentityProviderConfig;
 import uk.gov.ida.hub.config.domain.LevelOfAssurance;
-import uk.gov.ida.hub.config.domain.SignatureVerificationCertificate;
 
-import java.util.ArrayList;
 import java.util.List;
 
 
 public class IdentityProviderConfigDataBuilder {
 
     private String entityId = "default-idp-entity-id";
-    private List<SignatureVerificationCertificate> signatureVerificationCertificates = new ArrayList<>();
```

Comment:
It looks to me like this was redundant even before your changes - is that correct?

Reply:
Yes, it's been left over from the dark ages.